frontend/public/components/overview.jsx

```diff
@@ -559,15 +559,25 @@ export class Overview extends React.Component {
       </div>
       {
         !_.isEmpty(selectedItem) &&
-        <div className="overview__sidebar">
-          <div className="overview__sidebar-dismiss clearfix">
-            <CloseButton onClick={() => this.selectItem({})} />
-          </div>
-          <ResourceOverviewPage
-            kind={selectedItem.obj.kind}
-            resource={selectedItem.obj}
-          />
-        </div>
+        <Transition
```

Comment:
You could use `CSSTransition` to add the class names instead of doing it manually below.

https://reactcommunity.org/react-transition-group/css-transition